pilot/platform/cloudfoundry/config_test.go

```diff
@@ -0,0 +1,223 @@
+// Copyright 2017 Istio Authors
```

Comment:
We generally use standard go testing. I have used Ginkgo/Gomega in the past, but I find it is somewhat hard to read. But all the Gomega in this PR is in `platform/cloudfoundry`, so it doesn't effect other components.

Reply:
Yeah.. But its fine, as long as things are confined to this package.